seacrowd/sea_datasets/xm3600/xm3600.py

```diff
@@ -0,0 +1,192 @@
+    human-generated reference captions in 36 languages. The images were selected from across the world,
+    covering regions where the languages are spoken, and annotated with captions that achieve consistency in
+    terms of style across all languages, while avoiding annotation artifacts due to direct translation.
+    The languages covered in the dataset include Filipino, Indonesian, Thai, and Vietnamnese
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [SEACrowdConfig(name=f"xm3600_{lang}_source", version=datasets.Version(_SOURCE_VERSION), description=f"xm3600_{lang} source schema", schema="source", subset_id=f"xm3600_{lang}",) for lang in _LANGS] + [
+        SEACrowdConfig(
+            name=f"xm3600_{lang}_seacrowd_imtext",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"xm3600_{lang} SEACrowd schema",
+            schema="seacrowd_imtext",
+            subset_id=f"xm3600_{lang}",
+        )
+        for lang in _LANGS
+    ]
+
+    DEFAULT_CONFIG_NAME = "xm3600_id_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "image_paths": datasets.Value("string"),
+                    "texts": {
+                        "caption": datasets.Value("string"),
+                        "caption/tokenized": datasets.Value("string"),
+                        "caption/tokenized/lowercase": datasets.Value("string"),
+                    },
+                }
+            )
+        elif self.config.schema == "seacrowd_imtext":
+            features = schemas.image_text_features()
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        captions_path = dl_manager.download_and_extract(_URLS["captions"])
+        images_path = dl_manager.download_and_extract(_URLS["images"])
+        attr_path = dl_manager.download(_URLS["image_attributions"])
+
+        train_caps = {}
+        test_caps = {}
+        val_caps = {}
+
+        current_lang = self.config.subset_id.split("_")[1]
+
+        img_df = pd.read_csv(attr_path)
+
+        img_df_train = img_df.loc[img_df["Subset"] == "train"][["ImageID", "Subset"]]
+        img_df_test = img_df.loc[img_df["Subset"] == "test"][["ImageID", "Subset"]]
+        img_df_val = img_df.loc[img_df["Subset"] == "validation"][["ImageID", "Subset"]]
+
+        with jl.open(os.path.join(captions_path, "captions.jsonl"), mode="r") as j:
+            for l in j:
```

Comment:
No need to follow this suggestion, but anything to make it explicit:

```suggestion
        with jl.open(os.path.join(captions_path, "captions.jsonl"), mode="r") as file:
            for line in file:
```